### ISSUE TITLE
Docker for Mac - DockerHelper crashes constantly

### ISSUE THREAD

**Issue by NickLassondeAtLandmarkGlobal** [docker/for-mac#3470]
After upgrading to Docker for Mac 2.0.1.0 Edge, I repeatedly get a message that /Applications/Docker.app/Contents/Library/LoginItems/DockerHelper.app/Contents/MacOS/DockerHelper has crashed. Hitting "OK" dismisses the message, and it returns 10-60 seconds later. During this time, actual Docker images work fine.

  - [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: DD60E37F-FECD-45A7-B1F4-5742DE9C25BA/20190116182522

### Expected behavior

DockerHelper will not constantly crash.

### Actual behavior

DockerHelper does constantly crash.

### Information

  - macOS Version: 10.13.6

### Diagnostic logs

```
The entire diagnostics window says:
---
Diagnose succeded
---
Yes, "succeded" is misspelled in the window. 


```

### Steps to reproduce the behavior

  1. Upgrade to Docker 2.0.1.0 Edge
  2. Reboot
  3. Wait 5-60 seconds
  4. See the "DockerHelper quit unexpectedly" dialog
  5. Dismiss the dialog
  6. Go to 3.

**Comment by ii343hbka** [comment]
same for me. 
mpb 13" late 2011, 10.13.6, docker 2.0.1.0 Edge.

DockerHepler constantly fails generating lots of crashrepots. It utilises almost all processor time to do that.
lldb trace for DockerHelper:
`2019-01-20 13:32:59.281207+0300 DockerHelper[1347:67670] [] AppDelegate: launching...
Process 1347 stopped
* thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BAD_INSTRUCTION (code=EXC_I386_INVOP, subcode=0x0)
    frame #0: 0x0000000100010209 DockerHelper`DockerHelper.BundleConstants.init() -> DockerHelper.BundleConstants + 3177
DockerHelper`DockerHelper.BundleConstants.init() -> DockerHelper.BundleConstants:
->  0x100010209 <+3177>: ud2    
    0x10001020b <+3179>: xorps  %xmm0, %xmm0
    0x10001020e <+3182>: movaps %xmm0, -0x40(%rbp)
    0x100010212 <+3186>: movaps %xmm0, -0x50(%rbp)
Target 0: (DockerHelper) stopped.`

**Comment by znerd** [comment]
Same here. Diagnostics ID: `A08352A3-67AF-415B-8504-58349D2BF71C/20190121093728`

**Comment by pgayvallet** [comment]
Hello,

thanks for your report. We are reproducing the issue and are currently investigating this.

**Comment by pgayvallet** [comment]
The issue has been identified and fixed. This will be released with next edge.

**Comment by steakknife** [comment]
LGTM. No more crashtascularity. Fixed & #3504

**Comment by lukaszlach** [comment]
For me, it still constantly crashes on the newest Docker Desktop 2.0.1.0 (30090). It also utilizes 100% CPU all the time and makes Docker unusable.

`Diagnostic ID: DEDC9D9C-D9C1-4EA3-AB00-A178B2480C31/20190306155150`
